### ISSUE TITLE
add a persistent gauge type for usage with the tail plugin

### ISSUE THREAD

**Issue by gsauthof** [collectd/collectd#738]
When collection gauge numbers (i.e. DSType = GaugeLast) from a log file via the tail plugin, currently, there doesn't seem to be a way to specify:

> If nothing new got logged (to syslog and read by the tail plugin) in the current interval, just store the last known value.

This would be useful for tailing logs where the gauge number is only logged on changes, and the changes occur at a very low rate - e.g. with smartd:

```
Sep 18 00:20:04 example.org smartd[794]: Device: /dev/sda [SAT], initial Temperature is 40 Celsius (Min/Max ??/40)
Sep 18 00:37:55 example.org smartd[794]: Device: /dev/sda [SAT], Temperature changed -1 Celsius to 39 Celsius (Min/Max ??/40)
Sep 18 01:37:55 example.org smartd[794]: Device: /dev/sda [SAT], Temperature changed -1 Celsius to 38 Celsius (Min/Max 38!/40)
Sep 18 02:07:55 example.org smartd[794]: Device: /dev/sda [SAT], Temperature changed +1 Celsius to 39 Celsius (Min/Max 38/40)
Sep 18 02:37:55 example.org smartd[794]: Device: /dev/sda [SAT], Temperature changed -1 Celsius to 38 Celsius (Min/Max 38/40)
Sep 18 09:07:55 example.org smartd[794]: Device: /dev/sda [SAT], Temperature changed +1 Celsius to 39 Celsius (Min/Max 38/40)
Sep 18 09:37:55 example.org smartd[794]: Device: /dev/sda [SAT], Temperature changed -1 Celsius to 38 Celsius (Min/Max 38/40)
```

In the above example there may be several hours between a reported gauge number change.

Using the tail plugin as-is on that  would produce an virtually empty graph (mostly containing NAN values for the majority of 10-second-intervals).

See also the [plugin tail as source for smartcl tmp](http://mailman.verplant.org/pipermail/collectd/2010-April/003749.html) mailing list post from 2010 that discusses adding a new DSType `GaugeLastPersist`.

**Comment by janvonde** [comment]
Isn't this implemented in #2015?

**Comment by rpv-tomsk** [comment]
Yes, it looks like it was implemented. 
Closed.
